db/bigtable.go

```diff
@@ -938,6 +938,11 @@ func (bigtable *Bigtable) GetValidatorMissedAttestationsCount(validators []uint6
 				}
 			}
 		}
+
```

Comment:
I don't like this for at all (including your fix)

If startEpoch is 100.000 and endEpoch is 99.999, you are checking from 100.000 down to 1. In that case, endEpoch is not endEpoch, it's the amount of Epochs to be checked?

Reply:
Changed.